### DISCUSSION TITLE
umi 4 max模板logo怎么使用本地资源

### DISCUSSION THREAD

**Discussion by fnoopv** [umijs/umi#9902] [Q&A] [answered]
我看案例包括模板中全部都是url，怎么使用本地资源，比如svg?

官网中说logo支持Prolayout的全部配置，[链接](https://umijs.org/docs/max/layout-menu#%E4%BB%8B%E7%BB%8D)，但是prolayout时支持ReactNode的

**Comment by fnoopv** [comment]
找到了,`src/app.ts`改为`src/app.tsx`,但是路径就是问题了，会直接报错Module不存在

**Comment by fnoopv** [comment, marked as answer]
解决了,相对路径以`./`开头就行

**Comment by fz6m** [comment]
图片资源可以直接导入，获得他的 url ：

```ts
import imgUrl from './some-image.png'
```
